cardcomm/pkcs11/src/common/libtomcrypt/sha1.c

```diff
@@ -47,9 +47,9 @@ static int _sha1_compress(hash_state *md, unsigned char *buf)
 static int  sha1_compress(hash_state *md, unsigned char *buf)
 #endif
 {
-    ulong32 a,b,c,d,e,W[80],i;
+    unsigned long a,b,c,d,e,W[80],i;
```

Comment:
No. "unsigned long" is not the same thing as a 32-bit integer. There is a strong reason why this needs to be a 32-bit integer, rather than just "unsigned long".